identityAPI/post_identity_test.go

```diff
@@ -62,6 +61,33 @@ func TestCreateIdentitySuccess(t *testing.T) {
 	})
 }
 
+func TestCreateIdentity_EmailAlreadyAssociated(t *testing.T) {
+	identityAPI := httpexpect.New(t, cfg.IdentityAPIURL)
+
+	Convey("Given an email is already associated with an identity", t, func() {
+		id := createIdentity(identityAPI)
+
+		Convey("When a post request made to the API with the same email", func() {
+			body, err := json.Marshal(newIdentity)
+			So(err, ShouldBeNil)
+
+			resp := identityAPI.POST("/identity", nil).
+				WithHeader("Content-Type", "application/json").
+				WithBytes(body).
+				Expect()
+
+			Convey("Then a 400 status is returned", func() {
```

Comment:
Just a thought, is a 400 response the most informative option here? Would a ```409 Conflict``` be better?